pkgs/top-level/perl-packages.nix

```
@@ -17838,6 +17840,25 @@ let
       '';
   };
 
+  GraphicsMagick = buildPerlPackage {
+    pname = "GraphicsMagick";
+    version = "6.89-1";
+
+    buildInputs = [ pkgs.graphicsmagick.dev ];
+    outputs = [ "out" "dev" ]; # no devdoc
+    unpackPhase = ''
+      cp -r ${pkgs.graphicsmagick.dev} $out
```

```suggestion
      cp -r ${pkgs.graphicsmagick} $out
```

Wouldn't we miss files?